crowdsec-docs/docs/profiles/intro.md

```diff
@@ -4,11 +4,17 @@ title: Introduction
 sidebar_position: 1
 ---
 
-The profiles configuration allows user to configure which kind of remediation should be applied when a scenario is triggered. The profile can be used to :
- - increase or decrease decisions `duration` (default: `4h`)
- - decide the type of remediation that should be applied (default: `ban`)
- - decide of the scope of the remediation (default: `ip`)
- - enable/disable debug for a specific profile
+The profiles configuration allows users to configure which kind of remediation should be applied when a scenario is triggered. The profile can be used to:
 
+- increase or decrease decisions `duration` (default: `4h`)
+- decide the type of remediation that should be applied (default: `ban`)
+- decide the scope of the remediation (default: `ip`)
+- enable/disable debugging for a specific profile
```

Comment:
maybe we might want to reference notifications here (even if it's not related to your changes directly)